style.css

```diff
@@ -0,0 +1,1781 @@
+  left: 12px;
+}
+
+.lang {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: -20px;
+  gap: 12px;
+  padding: 0;
+}
+
+.lang li {
+  list-style-type: none;
+}
+
+.event18 {
+  box-sizing: border-box;
+  width: 327px;
+  height: 72px;
+  padding: 0 24px;
+  margin-left: -16px;
+  display: flex;
+  align-items: initial;
+  gap: 80px;
+  margin-top: -10px;
+  border-bottom: 1px solid rgb(223, 225, 230);
+}
+
+.event18 h3,
+.event17 h3 {
+  display: block;
+  height: 63.99px;
+  width: 112.11px;
+  margin-left: 18px;
+  box-sizing: border-box;
+  text-decoration: none;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+
+  /* display: flex; */
+  align-items: center;
+}
+
+.event17 a h3,
+.event18 a h3 {
+  top: calc(50% - 24px / 2);
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.separator {
+  width: 279px;
+  padding: 0 24px;
+  margin-left: 0;
+  margin-top: -10px;
+}
+
+.event-skill {
+  width: 295px;
+  height: 144px;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  align-items: flex-start;
+  margin: 0;
+  font-weight: bold;
+  margin-left: -25px;
+}
+
+.contact-derek {
+  background-color: #6070ff;
+  border-radius: 50px 0 0 0;
+}
+
+.contact-derek h2 {
+  color: white;
+  margin-top: 40px;
+  text-align: center;
+}
+
+.contact-derek p {
+  color: #fff;
+  margin-left: 10px;
+  margin-right: 10px;
+  text-align: center;
+}
+
+form input {
+  background: white;
+  padding: 8px;
+  border-radius: 8px;
+}
+
+form textarea {
+  background: hsl(0, 0%, 100%);
+  width: 200px;
+  height: 170px;
+  padding: 10px;
+  border: 1px solid hsl(198, 16%, 84%);
+}
+
+#username {
+  margin-left: 20px;
+  margin-right: 20px;
+  width: 80%;
+}
+
+#email {
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 80%;
+}
+
+#detail {
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 80%;
+}
+
+#btn {
+  border-radius: 8px;
+  background-color: white;
+  text-align: center;
+  color: #396df2;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  width: 30%;
+  height: 50px;
+}
+
+hr.rounded {
+  border-top: 8px solid black;
+  width: 30%;
+  border-radius: 5px;
+}
+
+.form-button {
+  box-sizing: border-box;
+  align-items: center;
+  margin-left: 20px;
+  width: 12vw;
+  height: 9vh;
+  left: 0;
+  top: calc(50% - 48px / 2);
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  color: #396df2;
+  transition: transform 0.4s ease-in-out;
+}
+
+#consection {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  justify-content: space-between;
```

Comment:
_Kindly remove one of this line. Unexpected duplicate "space-between"   declaration-block-no-duplicate-properties_

Reply:
Thank you so much!!!

I delete that.

justify-content: space-between;

Now is ok.

Have fun!